### PR TITLE
2 of 3 Separate Frames Viewer: Add View Mode Button to UI

### PR DESCRIPTION
**Pull request by goplayoutside3**
~~:warning: Includes #4452 please review that one first.~~

## Package
lib-classifier

## Linked Issue and/or Talk Post
Toward: https://github.com/zooniverse/front-end-monorepo/issues/4312

## Describe your changes
- Added ViewModeButton component. This button observes `separateFramesView` in the subject viewer store and renders an SVG icon accordingly.
- Added SeparateFramesViewer component. This new component will handle all separate frames view layout logic rather than directly in the flipbook viewer.
- Added stories and unit tests for new components.
- Added two new translations keys for ViewModeButton (will need follow-up in Lokalise)
- Handled the FieldGuideButton in each layout component

## How to Review

In the classifier storybook, checkout the ViewModeButton, SeparateFramesViewer, and SeparateFrame stories. Additionally, the FlipbookViewer has a story that renders the ViewModeButton in its controls. You can switch back and forth between flipbook and separate frames views in that story.

To view this feature in live projects:
    - Wildwatch Burrowing Owl is a flipbook project WITHOUT switching views enabled, so the ViewModeButton should not appear in its flipbook controls: https://local.zooniverse.org:3000/projects/sandiegozooglobal/wildwatch-burrowing-owl/classify?env=production
    - My test flipbook project does have switching views enabled, so you can see the ViewModeButton in its flipbook controls, and switch between flipbook and separate frames view while viewing this workflow: https://local.zooniverse.org:3000/projects/goplayoutside3/fem-flipbook-viewer/classify/workflow/22532?env=production

Design Review: @seanmiller26 here's some screenshots of the ViewModeButton for review!
flipbook

field guide

# Checklist
_PR Creator - Please cater the checklist to fit the review needed for your code changes._
_PR Reviewer - Use the checklist during your review. Each point should be checkmarked or discussed before PR approval._

## General
- [x] Tests are passing locally and on Github
- [x] Documentation is up to date and changelog has been updated if appropriate
- [x] You can `yarn panic && yarn bootstrap` or `docker-compose up --build` and FEM works as expected
- [ ] FEM works in all major desktop browsers: Firefox, Chrome, Edge, Safari (Use Browserstack account as needed)
- [ ] FEM works in a mobile browser

## General UX
Example Staging Project: [i-fancy-cats](https://local.zooniverse.org:3000/projects/brooke/i-fancy-cats)
- [x] All pages of a FEM project load: Home Page, Classify Page, and About Pages
- [x] Can submit a classification
- [x] Can sign-in and sign-out
- [ ] The component is accessible
  - Can be used with a screen reader [BBC guide to VoiceOver](https://bbc.github.io/accessibility-news-and-you/assistive-technology/testing-steps/voiceover-mac.html)
  - Can be used from the keyboard [WebAIM guide to keyboard testing](https://webaim.org/techniques/keyboard/#testing)
  - It is passing accessibility checks in the storybook

## New Feature
- [x] The PR creator has listed user actions to use when testing the new feature
- [x] Unit tests are included for the new feature
- [x] A storybook story has been created or updated
  - Example of SlideTutorial [component](https://github.com/zooniverse/front-end-monorepo/blob/master/packages/lib-classifier/src/components/Classifier/components/SlideTutorial/SlideTutorial.js) with docgen comments, and its [story](https://zooniverse.github.io/front-end-monorepo/@zooniverse/classifier/index.html?path=/docs/other-slidetutorial--default)

## Post-merge
- [ ] This PR adds translations keys to English dictionary(s). See guidance for pulling new keys to Lokalise [here](https://github.com/zooniverse/how-to-zooniverse/blob/master/Translations/lokalise.md#lokalise-and-fem).

### CODE REVIEW

**Comment by goplayoutside3** [comment]
Okay this is ready for review, and I updated the screenshots in the PR description.

**Comment by seanmiller26** [comment]
The ViewModeButton looks good! I have a few small notes:

- I think the icons need to be switched. The 'collapsed' view should show the icon with the photo in front to imply 'switch to separate photo frames' and the 'expanded' view should show the video icon to imply 'switch the gif/video view'.

- Is it possible for the last subject to have a blank toolbar for the icon to sit in? I believe it would look a bit more integrated.

- That brings up a semi-related question. Do we show tooltips for our tools on hover? If so I think the wording should be similar to above; 'switch to separate photo frames', 'switch the gif/video view'.

- A small issue for the Field Guide button: can we put the icon first, in front of the text? A little more research revealed: 
> In order for icons to serve as a visual scanning aid, users need to see them before they see the button label. Placing them to the left of your button label allows users to see the icon first. If you place icons to the right of your button label, they're not helping users scan because users see them last.

**Comment by seanmiller26** [comment]
> I recommend creating an Issue on Github and feel free to tag me in it!

Will do!

**Review by seanmiller26 — APPROVED** [review]
All the changes look good!

File: packages/lib-classifier/src/components/Classifier/components/FieldGuide/components/FieldGuideButton/FieldGuideButton.js [hunk 1]
```diff
@@ -41,20 +42,35 @@ export function ButtonLabel() {
...
     </Box>
   )
 }
 
-function FieldGuideButton({ fieldGuide = null, onClick = () => true, theme }) {
+function RowButtonLabel() {
+  const { t } = useTranslation('components')
+
+  return (
+    <Box direction='row' gap='8px' justify='center' align='center'>
```
Comment (mcbouslog) [inline packages/lib-classifier/src/components/Classifier/components/FieldGuide/components/FieldGuideButton/FieldGuideButton.js:54]: This is very minor, but the ColumnButtonLabel includes `as='span'`, not sure if we want to do the same with this label.

**Review by mcbouslog — APPROVED** [review]
Looking great! Really coming together.

The following isn't blocking at all, and we discussed with Sean in person more, but noting:
- it's a little disorienting when the ViewModeButton jumps from lower right below subject to lower left below last frame, not a big deal, but something to consider
- also a little confusing when the Field Guide button jumps from image toolbar to below task area, not sure how feasible, but maybe for workflows the can be separate frames (and video, text, others?) we keep Field Guide button below task area, including when in Flipbook mode for workflows that can be SeparateFrame

File: packages/lib-classifier/src/components/Classifier/components/FieldGuide/components/FieldGuideButton/FieldGuideButton.js (hunk 1 above)
Comment (goplayoutside3) [reply packages/lib-classifier/src/components/Classifier/components/FieldGuide/components/FieldGuideButton/FieldGuideButton.js:54]: Good catch! I'll update accordingly

**Pull request merged**